lib/logstash/filters/jdbc_static.rb

```diff
@@ -176,7 +180,9 @@ def loader_runner
   def prepare_data_dir
     # later, when local persistent databases are allowed set this property to LS_HOME/data/jdbc-static/
     # must take multi-pipelines into account and more than one config using the same jdbc-static settings
-    java.lang.System.setProperty("derby.system.home", ENV["HOME"])
+    path_data = Pathname.new(@settings.get_value("path.data")).join("plugins", "filter", "jdbc_static")
+    path_data.mkpath
+    java.lang.System.setProperty("derby.system.home", path_data.to_path)
```

Comment:
We don't have to copy safely these files, because on a restart, the `jdbc_static` filter, which uses an in-memory DB, has to recreate the Derby DB. The problem is that we left behind some trash. 
We have to clean it, but before have to find which files Derby creates.

Reply:
It creates a file named "derby.log", which in a local test contains:
```
andrea@kalispera ~ $ cat derby.log 
----------------------------------------------------------------
Mon Mar 14 12:00:24 CET 2022:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.1.0 - (1808820): instance a816c00e-017f-8816-1cb4-000051603af8 
on database directory memory:/home/andrea/c8f3ac67e8a5566f1a05a452 with class loader org.jruby.util.JRubyClassLoader@2da59753 
Loaded from file:/home/andrea/.m2/repository/org/apache/derby/derby/10.14.1.0/derby-10.14.1.0.jar
java.vendor=AdoptOpenJDK
java.runtime.version=11.0.11+9
user.dir=/home/andrea/workspace/logstash_configs/jdbc
os.name=Linux
os.arch=amd64
os.version=5.4.0-104-generic
derby.system.home=/home/andrea
Database Class Loader started - derby.database.classpath=''
```

Also described in https://db.apache.org/derby/docs/10.7/devguide/cdevdvlp27610.html.
We could remove it if we can, else warn the user we left a file behind to do a manual cleanup.

Reply:
Fixed by b4ccbb8070185d2580c28fb9f91a5a0c42f7910d